delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/FetchContext.scala

```diff
@@ -149,7 +149,7 @@ object FetchContext {
     apply(
       organizations.fetchActiveOrganization(_).void.toBIO[OrganizationRejection],
       projects.defaultApiMappings,
-      projects.fetch,
+      projects.fetch(_).toBIO[ProjectNotFound],
```

Comment:
Would this be safer as `ProjectRejection`?

Reply:
Just thinking if `fetch` was changed to raise something else, this could silently break route marshalling